addons/stock/models/product.py

```diff
@@ -371,8 +371,11 @@ def action_view_routes(self):
     @api.multi
     def write(self, values):
         res = super(Product, self).write(values)
-        if 'active' in values and not values['active'] and self.mapped('orderpoint_ids').filtered(lambda r: r.active):
-            raise UserError(_('You still have some active reordering rules on this product. Please archive or delete them first.'))
+        if 'active' in values and not values['active']:
+            orderpoints = self.mapped('orderpoint_ids').filtered(lambda r: r.active)
+            if orderpoints:
+                product = orderpoints[0].product_id
+                raise UserError(_('You still have some active reordering rules on this product : %s. Please archive or delete them first.') % product.display_name)
```

Comment:
@simongoffin my code is not the best (if there are more than one product), and like that the message is not modified (for translation).

```
if 'active' in values and not values['active']:
      products = self.mapped('orderpoint_ids').filtered(lambda r: r.active).mapped('product_id')
      if products:
           msg = _('You still have some active reordering rules on this product. Please archive or delete them first.')
           msg += '\n\n'
           for product in products:
                 msg += '- %s \n' % product.display_name
           raise UserError(msg)
```